staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -323,6 +323,12 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	resyncerrc := make(chan error, 1)
 	cancelCh := make(chan struct{})
 	defer close(cancelCh)
+	r.startResyncAsync(stopCh, cancelCh, resyncerrc)
```

Comment:
Actually, maybe let's switch to to:
```
go r.startResync(stopCh, cancelCh, resyncerrc)
```

and switch the startResync to get rid of `go() func() {}`

Reply:
done